src/main/java/ProjectOcean/Controller/SearchBrowseController.java

```diff
@@ -0,0 +1,44 @@
+package ProjectOcean.Controller;
+
+import ProjectOcean.Model.CoursePlanningSystem;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class SearchBrowseController extends AnchorPane {
+
+    @FXML
```

Comment:
Change to make @FXML annotation to be on the same row as the variable declaration